test/test_site/index.md

```diff
@@ -68,3 +68,14 @@ siteNav: site-nav.md
 <panel header="## Panel with src from another Markbind site header" src="sub_site/index.md" expanded>
 </panel>
 </div>
+
+# unexpanded panel
+<panel header="## unexpanded panel">
```

Comment:
Use `## unexpanded panel header` to avoid name clashing with the heading (one line above).